src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -897,6 +897,10 @@ reg_class v2_reg(
     V2, V2_H, V2_J, V2_K
 );
 
+reg_class v2_m2_reg(
```

Comment:
I don't know that `reg_class` expresses a "all of" rather than a "any of" set operation.